src/test/java/Homework19.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+@Test
+public void deletePlaylist() throws InterruptedException {
+login("guyvanalex4@gmail.com", "teststudent!1");
+playlist();
+verifyPlaylistDeleted();
+Thread.sleep(2000);
```

Comment:
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.